sdk/search/search-documents/src/base64.browser.ts

```diff
@@ -3,15 +3,15 @@
 
 /**
  * Encodes a string in base64 format.
- * @param value The string to encode.
+ * @param value - The string to encode.
```

Comment:
huh, I see TSDoc wants this, but I haven't seen it before now. Is this a recent change they made or did they just grandfather in the old JSDoc format?

Reply:
not sure about the motivation behind this syntax but this is supposedly the origin: https://github.com/microsoft/tsdoc/issues/8

Reply:
Digging through that thread, I get the idea from here: https://github.com/microsoft/tsdoc/issues/8#issuecomment-424751850

That the only real reason is that TypeDoc had implemented `@param` in a way that has an unnecessary hyphen and they kept that behavior.